Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/type/BaseType.java

```diff
@@ -39,14 +45,46 @@
     }
 
     /**
+     * Method for adding relationship between bean objects.
+     * 
+     * @param objects
+     *            list
+     * @return JSONArray
+     */
+    @SuppressWarnings("unchecked")
+    <F extends BaseBean> JSONArray addObjectRelation(List<F> objects) {
+        JSONArray jsonArray = new JSONArray();
+        if (objects != null) {
```

Comment:
There are different styles of reading code. For this code lines it does not matter.